l10n_ch_dta/bank_view.xml

```diff
@@ -6,11 +6,9 @@
       <field name="model">res.partner.bank</field>
       <field name="inherit_id" ref="base.view_partner_bank_form"/>
       <field name="arch" type="xml">
-        <group name="bank" position="after">
-          <group string="DTA options" attrs="{'invisible': [('company_id', '=', False)]}">
```

Comment:
company_id = False meant that we wanted to display the config only for the bank of the company and not on the bank of the suppliers or customers.

Is there no way to reproduce this ?